lumino-explorer-api/src/main/java/org/rif/lumino/explorer/models/documents/LuminoHub.java

```diff
@@ -0,0 +1,58 @@
+package org.rif.lumino.explorer.models.documents;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document("lumino_hub")
+public class LuminoHub {
+
+    public static final int MAX_CONNECTIONS = 50;
```

Comment:
I think it would be better to move this to a system property, because in a future we will modify this value.